xml/Country.xml

```diff
@@ -1131,6 +1145,13 @@
                 <narrative xml:lang="fr">Antilles néerlandaises (les )</narrative>
             </name>
         </codelist-item>
+        <codelist-item status="withdrawn">
+            <code>NT</code>
+            <name>
+                <narrative>Neutral Zone</narrative>
+                <narrative xml:lang="fr">Zone neutre</narrative>
+            </name>
+        </codelist-item>
```

Comment:
This seems like a funny one: https://en.wikipedia.org/wiki/Saudi_Arabian%E2%80%93Iraqi_neutral_zone

I guess we should be consistent with including all (or no) transitionally reserved country codes though. What do you think?

Reply:
Yeah, weird, but I reckon include it.

Reply:
OK sounds good!